### PR TITLE
Update navigation steps - Notifications

### PR DESCRIPTION
**Pull request by ThatElfGuy**
Updating and clarifying navigation steps

### CODE REVIEW

File: 02_profiles/09_email-notifications.md [hunk 1]
```diff
@@ -4,9 +4,11 @@ You can update your email notification settings in either of two ways.
...
 Your global email notifications settings are in your "Update Settings" area of your profile:
 
-1. Click your nickname in the upper-right corner of FetLife and select "Update Settings" from the dropdown menu.
-2. The "Email Notifications" tab has all of your email notifications in one place. Click On or Off as appropriate.
-3. Be sure to press the "Update" button at the bottom of the screen to save your changes!
+1. Click on the hamburger menu (icon with three horizontal lines) on the far right of FetLife's navigation bar. 
+2. Click on “Update Settings" in the pop out menu.
+3. Click on the "Notifications" tab. 
+4. Check/uncheck each box appropriately.
+
```
Comment (ThatElfGuy) [inline 02_profiles/09_email-notifications.md]: I think everything below this can be removed, honestly.

Comment (Pairadox) [reply 02_profiles/09_email-notifications.md]: I disagree - it explains what might otherwise be seen as confusing.

Comment (jminion) [reply 02_profiles/09_email-notifications.md]: I think those should be there also.

**Comment by EGypsy** [comment]
agree with @Pairadox - I think keeping that info, even if it seems a bit tedious, is helpful.

**Pull request merged**